.goreleaser-alpha.yml

```diff
@@ -0,0 +1,63 @@
+# Visit https://goreleaser.com for documentation on how to customize this
+# behavior.
+version: 2
+before:
+  hooks:
+    # this is just an example and not a requirement for provider building/publishing
+    - go mod tidy
+builds:
+  - env:
+      # goreleaser does not work with CGO, it could also complicate
+      # usage by users in CI/CD systems like Terraform Cloud where
+      # they are unable to install libraries.
+      - CGO_ENABLED=0
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    flags:
+      - -trimpath
+    ldflags:
+      - '-s -w -X github.com/ClickHouse/terraform-provider-clickhouse/pkg/project.version={{.Version}} -X github.com/ClickHouse/terraform-provider-clickhouse/pkg/project.commit={{.Commit}}'
+    goos:
+      - freebsd
+      - windows
+      - linux
+      - darwin
+    goarch:
+      - amd64
+      - '386'
+      - arm
+      - arm64
+    ignore:
+      - goos: darwin
+        goarch: '386'
+    tags:
```

Comment:
this is a copy of the original config file, with only this section changed

Reply:
Maybe let's add this comment right into file header? Otherwise it is not quite clear why we having two almost identical files. 

By the way, I thought it was possible with goreleaser to have 2 build targets inside one file. Then you can use Yaml anchors/aliases to deduplicate

Reply:
Feel free to merge this anyway, if we need it before 2.0.0 release. We can always de-duplicate later

Reply:
it requires goreleaser pro license